ProteoformSuiteInternal/Proteoform.cs

```diff
@@ -168,14 +183,15 @@ public List<ExperimentalProteoform> identify_connected_experimentals(List<PtmSet
                         rank_sum += mod_rank;
```

Comment:
@acesnik When would this be the case? Are there mods with monoisotopic mass of 0?